scripts/build/daedalus-bridge.sh

```diff
@@ -20,3 +20,6 @@ echo "3. Building Bridge..."
 cd daedalus
 npm install
 npm run build:prod
+
+echo "4. Linking Bridge..."
+npm link
```

Comment:
As for me, I'm glad that `npm link` was absent here, because it fails for me when done without `sudo`, while it doesn't for others.
However I have little authority to decide on this

Reply:
Hmm well if that's the case for many people, maybe adding a note to the documentation would be better.

Reply:
I also believe adding note should be better. There was a separate script that does:
```
daedalus-bridge.sh
npm link
```
which was used just for this purpose, but I don't know when and why it was removed.

@DominikGuzei @darko-mijic do you run `npm link` explicitly or do you use some script from daedalus repo that handles this?

Instead of this proposed solution I would create daedalus-bridge-link.sh that does what is described above.